### DISCUSSION TITLE
Account catalogue for service accounts

### DISCUSSION THREAD

**Discussion by dynamic-entropy** [dmwm/CMSRucio#560] [Ideas]
At the moment service accounts are created and managed manually. 

- If we want to continue operating them by hand, we can close this
- should there be a sync from CRIC or any other source?

Triggered by: https://its.cern.ch/jira/browse/CMSTRANSF-655

@ericvaandering @dciangot comments?

**Comment by dciangot** [comment]
Not sure how this can be managed at cric level, but whatever option looks better than the manual one to me.
